### PR TITLE
Move the creation of the QR code to a js file.

### PR DESCRIPTION
**Pull request by LaughingJohn**
In-line JavaScript is insecure and breaks script-src 'unsafe-inline' in Content Security Policy.

Note: I think the part about locating the Scripts section in EnableAuthenticator.cshtml should probably go after bit I've added about creating the js file (qr.js), and before updating the scripts section. I haven't moved it as I wasn't sure how the "moniker" directives worked and didn't want to break anything. (my first ever documentation proposal)

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=23306) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=23306) before we can accept your contribution.<br/><br/>:x: LaughingJohn [sign now](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=23306)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/AspNetCore.Docs?pullRequest=23306) it.</sub>

File: aspnetcore/security/authentication/identity-enable-qrcodes.md [hunk 1]
```diff
@@ -52,22 +52,28 @@ These instructions use *qrcode.js* from the https://davidshimjs.github.io/qrcode
     @await Html.PartialAsync("_ValidationScriptsPartial")
 }
 ```
+* Create a new JavaScript file called `qr.js` in wwwroot/js and add the following code to generate the QR Code:
```
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/identity-enable-qrcodes.md:55]:
```suggestion
* Create a new JavaScript file called *qr.js* in *wwwroot/js* and add the following code to generate the QR Code:
```

File: aspnetcore/security/authentication/identity-enable-qrcodes.md [hunk 2]
```diff
@@ -52,22 +52,28 @@ These instructions use *qrcode.js* from the https://davidshimjs.github.io/qrcode
...
+        {
+            text: uri,
+            width: 150,
+            height: 150
+        });
+});
+```
 
-* Update the `Scripts` section to add a reference to the `qrcodejs` library you added and a call to generate the QR Code. It should look as follows:
+* Update the `Scripts` section to add a reference to the `qrcode.js` library you downloaded previously and also add the `qr.js` file with the call to generate the QR code. Based on our example it should look as follows:
```
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/identity-enable-qrcodes.md:69]:
```suggestion
* Update the `Scripts` section to add a reference to the `qrcode.js` library previously downloaded.
* Add the *qr.js* file with the call to generate the QR code:
```

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@LaughingJohn  thanks for doing this. See my commit for slight edit I made to conform to our style guide.

**Pull request merged**